app/lib/utils/searchAndFiltersHelpers.ts

```diff
@@ -0,0 +1,20 @@
+import { escapeRegex } from './escapeRegex';
+
+export const searchHelper = (search?: string) => {
+  if (!search) return '';
```

Comment:
maybe it will be better to return null for example instead of empty string? since we return an array in the end
```suggestion
  if (!search) return null;
```

Reply:
empty string is coerced in the if statement , so if there is no serach statement in wont run in both ways , but i agree with the semantic clarity, good catch